charts/kof-mothership/templates/kcm/service-monitor/kcm-controller-manager.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Let's wrap it under the condition, like [this](https://github.com/opencost/opencost-helm-chart/blob/d4956d96d86bee9fa35b8730b4872aae8c9158d7/charts/opencost/templates/servicemonitor.yaml#L1), but keep it enabled by default.

```
{{- if .Values.kcm.serviceMonitor.enabled -}}
```

Reply:
Let's add a promxy filter to the dashboard like [this](https://github.com/gmlexx/kof/blob/8f0646c3ec11eae1a5e14add348c71444e42d5d5/charts/kof-mothership/files/dashboards/kubernetes-views-global.yaml#L2562) and add PromxyServerGroup deployment to fetch data via promxy.